src/PointwiseFunctions/AnalyticSolutions/NewtonianEuler/IsentropicVortex.hpp

```diff
@@ -154,8 +153,7 @@ class IsentropicVortex : public MarkAsAnalyticSolution {
                    const std::array<double, Dim>& center,
                    const std::array<double, Dim>& mean_velocity,
                    double strength,
-                   double perturbation_amplitude =
-                       PerturbAmplitude::default_value()) noexcept;
+                   double perturbation_amplitude = 0.0) noexcept;
 
```

Comment:
It seems to me @carmaza set the default value to `PerturbAmplitude::default_value()`. I wonder whether that was to better illustrate the dependencies, although these changes make it more consistent I think.

Reply:
I can't think of why I did this, but the change suggested here seems fine. Setting `perturbation_amplitude`  to 0.0 reproduces the 2-d case immersed in 3-d, while a nonzero value will activate a source term only present in 3-d.